packages/protocol/contracts/common/UsingPrecompiles.sol

```diff
@@ -123,16 +177,20 @@ contract UsingPrecompiles {
 
   /**
    * @notice Checks a BLS proof of possession.
-   * @param proofOfPossessionBytes The public key and signature of the proof of possession.
+   * @param sender The address signed by the BLS key to generate the proof of possession.
+   * @param blsKey The BLS public key that the validator is using for consensus, should pass proof
+   *   of possession. 48 bytes.
+   * @param blsPop The BLS public key proof-of-possession, which consists of a signature on the
+   *   account address. 96 bytes.
    * @return True upon success.
    */
-  function checkProofOfPossession(address sender, bytes memory proofOfPossessionBytes)
+  function checkProofOfPossession(address sender, bytes memory blsKey, bytes memory blsPop)
```

Comment:
I'm very happy you were able to separate it into two values! Previously, this caused the contract to have more than the maximum code size. How did you get around it?

Reply:
We upped the max code size 😄

Reply:
:D